runners/google-cloud-dataflow-java/worker/build.gradle

```diff
@@ -0,0 +1,215 @@
+    include(dependency(".*:.*"))
+
+    if (is_legacy_worker()) {
+      sdk_provided_dependencies.each {
+        exclude(dependency(it))
+      }
+      excluded_dependencies.each {
+        exclude(dependency(it))
+      }
+    }
+  }
+
+  // Archive name pattern: ${name}-${appendix}-${version}-${classifier}.jar
+  appendix = is_legacy_worker() ? "legacy-bundled" : "fnapi-bundled"
+
+  // Include original source files extracted under
+  // '$buildDir/original_sources_to_package' to jar
+  from "$buildDir/original_sources_to_package"
+
+  exclude "META-INF/LICENSE.txt"
+  exclude "about.html"
+  exclude "google/protobuf/*.proto"
+  exclude "windmill*.proto"
+
+  if (is_legacy_worker()) {
+    relocate("com.", getWorkerRelocatedPath("com.")) {
+      exclude "com.fasterxml.jackson.**"
+      exclude "com.google.api.client.**"
+      exclude "com.google.api.services.bigquery.**"
+      exclude "com.google.api.services.clouddebugger.**"
+      exclude "com.google.api.services.dataflow.**"
+      exclude "com.google.api.services.datastore.**"
+      exclude "com.google.api.services.pubsub.**"
+      exclude "com.google.api.services.storage.**"
+      exclude "com.google.auth.**"
+      exclude "com.google.cloud.dataflow.**"
+      exclude "com.sun.management*"
+      exclude "com.sun.management.**"
+    }
+    relocate("javax.servlet", getWorkerRelocatedPath("javax.servlet"))
+    relocate("io.", getWorkerRelocatedPath("io."))
+    relocate("okio.", getWorkerRelocatedPath("okio."))
+    relocate("org.", getWorkerRelocatedPath("org.")) {
+      // Exclude netty-tcnative from shading since gRPC relies on Netty to be able
+      // to load org.apache.tomcat.jni.SSL to provide an SSL context.
+      exclude "org.apache.avro.**"
+      exclude "org.apache.beam.**"
+      exclude "org.apache.tomcat.jni.**"
+      exclude "org.conscrypt.**"
+      exclude "org.eclipse.jetty.alpn.**"
+      exclude "org.eclipse.jetty.npn.**"
+      exclude "org.hamcrest.**"
+      exclude "org.joda.time.**"
+      exclude "org.junit.**"
+      exclude "org.slf4j.**"
+      exclude "org.w3c.dom.**"
+    }
+    relocate("org.apache.beam.runners.core.construction.",
+             getWorkerRelocatedPath("org."))
+  }
+})
+
+/******************************************************************************/
+// Configure the worker root project
+
+configurations {
+  sourceFile
+
+  // Ban these dependencies from all configurations
+  all {
+    // Ban the usage of AppleJavaExtensions in findbugs.
+    exclude group: "com.apple", module: "AppleJavaExtensions"
+  }
+}
+
+dependencies {
+  // Set dependencies to shadow scope by default, but in a property so they can
+  // be downgraded when building a legacy (non-FnAPI) worker.
+  if (is_legacy_worker()) {
+    sdk_provided_dependencies.each {
+      provided(it)
+    }
+  } else {
+    sdk_provided_dependencies.each {
+      shadow(it)
+    }
+  }
+
+  compile "org.apache.beam:beam-model-fn-execution:$version"
```

Comment:
Won't this pull the dependency from maven rather than try and build what's in this repository? (I could be missunderstanding things, but it seems we want compile project(...) hear instead.)

Reply:
Thanks @robertwb. I think you are right. I'll create a new PR to fix this build.